packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/FishjamClientInternal.kt

```diff
@@ -102,6 +103,18 @@ internal class FishjamClientInternal(
           commandsQueue.clear("Websocket was closed")
         }
 
+        override fun onClosing(
+          webSocket: WebSocket,
+          code: Int,
+          reason: String
+        ) {
+          if (AuthError.isAuthError(reason)) {
+            listener.onAuthError(AuthError.fromString(reason))
+          }
+          commandsQueue.clear("Websocket was closed, reason: $reason")
```

Comment:
Maybe instead of fixed string lets use enum with code, reason and text message.

Reply:
`commandsQueue.clear` no longer throws any exceptions, just ends jobs. Errors in android-client are handled by callbacks like `onSocketError`, `onAuthError` etc.
In rn-client on Android side I've added errors as classes (inspired by expo).
We could try something like this also on iOS side but with ios refactor ongoing it would be a mess.